indico/modules/events/abstracts/placeholders.py

```diff
@@ -105,9 +105,13 @@ class ContributionTypePlaceholder(Placeholder):
 
     @classmethod
     def render(cls, abstract):
-        if abstract.state == AbstractState.accepted:
-            return abstract.accepted_contrib_type.name if abstract.accepted_contrib_type else ''
-        return ''
+        if abstract.state == AbstractState.withdrawn:
+            ctype = abstract.accepted_contrib_type or abstract.submitted_contrib_type
+        elif abstract.state != AbstractState.accepted:
```

Comment:
This kind of logic is a bit confusing. I initially read that `!=` as `==` and it didn't make sense, of course. In cases where there's `if... else...` I normally favour using the `if` for the "positive" case.